week-02/assessment/gomoku/src/main/java/learn/gomoku/GameController.java

```diff
@@ -0,0 +1,131 @@
+package learn.gomoku;
+
+import learn.gomoku.game.Gomoku;
+import learn.gomoku.game.Result;
+import learn.gomoku.game.Stone;
+import learn.gomoku.players.HumanPlayer;
+import learn.gomoku.players.Player;
+import learn.gomoku.players.RandomPlayer;
+
+import java.util.List;
+import java.util.Scanner;
+
+import static learn.gomoku.game.Gomoku.WIDTH;
+
+public class GameController {
+
+    private Scanner console = new Scanner(System.in);
+    Gomoku game;
+
+
+    public void run(){
+        while(playAgain()) {
```

Comment:
MAJOR: not quite right as now you're asking them to play "again" before the first iteration.  change this to a do/while with a single call to playAgain() as the while condition